### PR TITLE
[IMP] sale_timesheet: add test related to the uom onchange

### PR DESCRIPTION
**Pull request by hugocarlier**
Previous test performed with a tour failed underterministicaly. It is
therefore replaced by a test method in the test class TestSaleTimesheet.

related-#125023
related-#105951

### CODE REVIEW

**Review by xavierbol — APPROVED** [review]
LGTM, thanks for your work, I will let the runbot finishes its build and then I will merge it. 🙂 

robodoo rebase-ff

File: addons/sale_timesheet/static/tests/tours/sale_timesheet_tour.js [hunk 1]
```diff
@@ -89,10 +24,10 @@ registry.category("web_tour.tours").add('sale_timesheet_tour', {
     content: 'Click on "Add a product" to add a new product. We will add a service product.',
 }, {
     trigger: '.o_field_html[name="product_id"], .o_field_widget[name="product_template_id"] input',
-    content: 'Select a prepaid service product (e.g. Service Product Prepaid Hours)',
-    run: 'text Service Product Prepaid Hours',
+    content: Markup('Select a prepaid service product <i>(e.g. Service Product (Prepaid Hours))</i>'),
```
Comment (xavierbol) [inline addons/sale_timesheet/static/tests/tours/sale_timesheet_tour.js:27]: Hello @odoo/rd-security ,

Can we have an override for this PR? 

We use the Markup function with a static content in a JS tour used in a python test. 🙂 

Thanks in advance!

Have a nice day! 🙂

Comment (xavierbol) [reply addons/sale_timesheet/static/tests/tours/sale_timesheet_tour.js:27]: Coucou @mart-e 👋🏼 

Can we have an security override for this PR?
We use Markup function with a static content in a JS tour used in a python test. 🙂 

Thanks in advance! 

Have a nice afternoon! 🙂

**Comment by mart-e** [comment]
@robodoo override=ci/security
Reason: static string

**Comment by xavierbol** [comment]
robodoo r+

**Pull request closed without merging**